dspace-api/src/main/java/org/dspace/eperson/dao/EPersonDAO.java

```diff
@@ -38,7 +38,29 @@ public List<EPerson> search(Context context, String query, List<MetadataField> q
 
     public int searchResultCount(Context context, String query, List<MetadataField> queryFields) throws SQLException;
 
-    public List<EPerson> findByGroups(Context context, Set<Group> groups) throws SQLException;
+    /**
+     * Find all EPersons who are a member of one or more of the listed groups in a paginated fashion. Order is
+     * indeterminate.
```

Comment:
I think you are correct here. I think the order *is* consistent, but it may order by UUID.  I'll dig into this more and make sure to fix the docs and/or code here.

Reply:
I've verified this comment is incorrect.  The default ordering of findByGroups() is by EPerson UUID.  I didn't modify this behavior in this PR, but have corrected the comment in https://github.com/DSpace/DSpace/pull/9078/commits/09e7f105e063ffa7fa35abe7cd9160c59b0f91a8.  This PR just added pagination to findByGroups() without modifications to the underlying query itself.